### PR TITLE
fixed mask_color

### LINKED ISSUE #3007: "Masking the solar disk" gallery example does not have a black mask
### Description
In the "Masking the solar disk" example, the mask is incorrectly colored. The mask is explicitly set to be black
```python
palette = aia.plot_settings['cmap']
palette.set_bad('black')
```
but it appears white in the generated gallery example. I suspect this is due to a change in the way matplotlib treats NaNs in imshow.

### Steps to Reproduce
See example gallery in docs.

### PR DESCRIPTION
**Pull request by ravindra1307**
### Description

Fixes #3007<Issue Number>

### CODE REVIEW

File: examples/computer_vision_techniques/mask_disk.py [hunk 1]
```diff
@@ -41,7 +42,7 @@
 mask = ma.masked_less_equal(r, 1)
 palette = aia.plot_settings['cmap']
 palette.set_bad('black')
-
+norm = mpl.colors.Normalize(vmin=-1000, vmax=3000)
```
Comment (nabobalis) [inline examples/computer_vision_techniques/mask_disk.py]: Can you add a comment here as to why we had to add this line?

Comment (ravindra1307) [reply examples/computer_vision_techniques/mask_disk.py]: i have added comment, i have tried for this image only for others i think we have to experiment with different values

File: examples/computer_vision_techniques/mask_disk.py [hunk 2]
```diff
@@ -42,6 +43,9 @@
 palette = aia.plot_settings['cmap']
 palette.set_bad('black')
 
+###############################################################################
```
Comment (nabobalis) [inline examples/computer_vision_techniques/mask_disk.py]: I don't feel this comment explains enough, why is it negative, why this range and etc.

File: examples/computer_vision_techniques/mask_disk.py (hunk 1 above)
Comment (nabobalis) [reply examples/computer_vision_techniques/mask_disk.py]: That's a problem. We can't expect our users to have to manually change the norm in some (what feels like random) way to get the mask to work. It's ok for this example but I don't think we can count it as a fix for the original issue.

**Comment by nabobalis** [comment]
Sorry again @ravindra1307 but having looked at this again, I don't think that making this change is a good idea. 

We need to change the underlying setting somehow instead of asking our users to change the normalization like that.

**Pull request closed without merging**